apps/argparse/argparse.hpp

```diff
@@ -701,7 +701,14 @@ class Argument {
   template <typename T, typename std::enable_if<std::is_integral<T>::value>::type * = nullptr>
   auto &store_into(T &var) {
     if (m_default_value.has_value()) {
-      var = std::any_cast<T>(m_default_value);
+      try
```

Comment:
have those changed been upstreamed ?

Reply:
No.

Reply:
would be good to do, to easy later resync